### PR TITLE
Chute deploying inventory

### LINKED ISSUE #4317: [MC1.16.1 | IE108] Chutes can't be dropped into as inventories
This means that you can't drop directly into them using hoppers, dropping conveyors, item routers, etc. (as a fun fact, it DOES work with droppers though, even through the sides of a chute \^\^)

### PR DESCRIPTION
**Pull request by MalkContent**
gave chutes conveyor like deploying inventories
threw in a typo fix for conveyor magnet supression -> suppression

fixes: https://github.com/BluSunrize/ImmersiveEngineering/issues/4317

### CODE REVIEW

**Comment by TeamSpen210** [comment]
Changing the API package's methods would cause a bit of an unnecessary compatibility break, would probably be better to keep the old names as an alias.

**Comment by MalkContent** [comment]
fair point
I'll make it its own commit

**Comment by BluSunrize** [comment]
Yeah please remove everything related to magnets from this. We can't change API stuff on the fly, that would demand an API version increase.
It gets a chunky todo of "Do this next time we bump API version" .... and then we forget about it next time we bump the API version. x3

**Comment by MalkContent** [comment]
hah ^^ well it's rolled back now
the chute stuff itself never contained anti magnet stuff, besides the player-pickup prevention on deploying the item, which was already present in other parts of the chute's code

File: src/main/java/blusunrize/immersiveengineering/common/blocks/metal/ChuteTileEntity.java [hunk 1]
```diff
@@ -384,4 +388,72 @@ public boolean receiveClientEvent(int id, int arg)
...
+	@Override
+	public <T> LazyOptional<T> getCapability(@Nonnull Capability<T> cap, @Nullable Direction side)
+	{
+		if(cap==CapabilityItemHandler.ITEM_HANDLER_CAPABILITY&&side==Direction.UP)
+			return insertionCap.cast();
+		return super.getCapability(cap, side);
+	}
+
+	public static class ChuteInventoryHandler implements IItemHandlerModifiable
+	{
```
Comment (BluSunrize) [inline src/main/java/blusunrize/immersiveengineering/common/blocks/metal/ChuteTileEntity.java:402]: This doesn't need to be Modifiable. Make it `IItemHandler` instead and ditch the `setStackInSlot` method

**Review by BluSunrize — CHANGES_REQUESTED** [review]
Minor change request

File: src/main/java/blusunrize/immersiveengineering/common/blocks/metal/ChuteTileEntity.java (hunk 1 above)
Comment (MalkContent) [reply src/main/java/blusunrize/immersiveengineering/common/blocks/metal/ChuteTileEntity.java:402]: done. This was almost a direct copy of ConveyorInventoryHandler. I don't know if that could be changed, too, or if there was a good reason for it to be IItemHandlerModifiable

**Review by malte0811 — APPROVED** [review]

**Review by BluSunrize — APPROVED** [review]
LGTM

**Pull request merged**